portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,263 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+* {
+    font-family: 'Lato', serif;
 }
 
-#greeting-container {
-  margin-top: 20px;
+body {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
 }
+
+main section.home {
+    padding-top: 60px;
+    margin-top: 0px;
+}
+
+main section {
+    margin: 10px 0 10px 0;
+}
+
+a {
+    color: #22C9E0;
+    text-decoration: none;
+}
+
+a:hover, a:focus, a:active {
+    text-decoration: underline;
+}
+
+/* navigation bar */ 
+
+nav {
+    background-color: #22C9E0;
+    position: fixed;
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    align-items: center;
+}
+
+.nav__logo {
+    padding-left: 50px;
+    padding-right: 50px;
+}
+
+.nav__items {
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    list-style-type: none;
+    margin: 0;
+    padding: 0 100px 0 0;
+    width: 100%;
+}
+
+.nav__item {
+    padding: 0 25px 0 25px;
+    font-size: 17pt;
+    color: white;
+    border-bottom: 1px solid transparent;
+    padding-bottom: 1px;
+}
+
+.nav__item:hover, .nav__item:focus {
+    border-bottom: 2px solid white;
+    padding-bottom: 1px;
+}
+
+/* home section */
+
+section.home {
```

Comment:
From our [style guide](https://google.github.io/styleguide/htmlcssguide.html#Type_Selectors), it's not recommended to use type selectors. Maybe change `section.home` to `.home`? Same for other occurrences.